lib/web_ui/test/paragraph_test.dart

```diff
@@ -406,9 +406,9 @@ void main() async {
     expect(
       paragraph.getBoxesForRange(1, 2).single,
       const TextBox.fromLTRBD(
-        10,
```

Comment:
reason for change?

Reply:
Fixed a bug by making paragraph style attributes non-null. Notice that the test uses RTL, so the box should have been pushed to the right, but previously it wasn't. The change removed a bunch of null-checks one of which used a wrong value.